src/components/HomeHero.vue

```diff
@@ -5,9 +5,16 @@
 </template>
 
 <style scoped>
-  .hero {
-    min-height: 400px;
-    padding: 80px 0;
-    background: linear-gradient(to right, #5eade3 0%,#c340b9 100%);
+.hero {
+  min-height: 550px;
```

Comment:
Let's increase this to `600px` given the new font and layout.
The `min-height` is to stop jumping when changing between new/edit modes.

Reply:
fixed